libr/socket/socket.c

```diff
@@ -678,19 +654,10 @@ R_API int r_socket_flush(RSocket *s) {
 	return true;
 }
 
-// XXX: rewrite it to use select //
 /* waits secs until new data is received.	  */
 /* returns -1 on error, 0 is false, 1 is true */
 R_API int r_socket_ready(RSocket *s, int secs, int usecs) {
-#if __UNIX__
-	//int msecs = (1000 * secs) + (usecs / 1000);
-	int msecs = (usecs / 1000);
-	struct pollfd fds[1];
-	fds[0].fd = s->fd;
-	fds[0].events = POLLIN | POLLPRI;
-	fds[0].revents = POLLNVAL | POLLHUP | POLLERR;
-	return poll ((struct pollfd *)&fds, 1, msecs);
-#elif __WINDOWS__
+#if __UNIX__ || __WINDOWS__
```

Comment:
isnt this the same as `#if 1` ? :D i dont think we cover more platforms than unix (variants) and windows. just remove the #if block